kuma/contributions/jinja2/contributions/contribute.html

```diff
@@ -48,7 +48,7 @@ <h3>{{ title }}</h3>
         <div class="index">
             <h2>{{_('FAQs')}}</h2>
         </div>
-        <ol class="faqs clear">
+        <ol id="contribute-faqs" class="faqs clear">
```

Comment:
👍 "Learn more about payments to MDN" link from the banner now goes to the first FAQ entry